source/guide/3.2/plugin-softlayer.md

```diff
@@ -0,0 +1,468 @@
+---
+layout: bt_wiki
+title: SoftLayer Plugin
+category: Plugins
+publish: true
+abstract: Cloudify SoftLayer plugin description and configuration
+pageord: 600
+---
+
+
+{%summary%}
+This section describes how to use SoftLayer based cloud infrastructure in your services and applications.
+For more information about SoftLayer, please refer to: [http://www.softlayer.com/](http://www.softlayer.com/).
+{%endsummary%}
+
+
+# Types
+
+## cloudify.softlayer.nodes.VirtualServer
+
+
+**Derived From:** [cloudify.nodes.Compute](reference-types.html)
+
+
+**Properties:**
+
+  * **Required Properties:**
+    * `location` The data center.
+    * `domain` The domain of the server.
+    * `ram` The item id of the desired server's RAM, e.g. item id 864 for 8 GB.
+    * `cpu` The item id of the desired server's CPU, e.g. item id 859 for 4 X 2.0 GHz Cores.
+    * `disk` The item id of the desired server's first disk, e.g. item id 1178 for 25 GB (SAN).
+
+  * **Optional Properties:**
+    * `api_config`  A dictionary of SoftLayer Portal username, API Key and endpoint URL of choice.
+      * An empty dictionary by default - will be taken from other resources if not specified - see [SoftLyaer authentication](#softlayer-authentication))
+    * `os` The item id of the operating system to use, e.g. item id 1857 for Windows Server 2008 R2 Standard Edition (64bit)
+    * `image_template_global_id` An image template global id to load the server with. 
```

Comment:
I loved your previous e.g.'s. Maybe have those for the rest of the properties? It really helps.